components/scanners/trufflehog/internal/transformer/transformer.go

```diff
@@ -171,10 +172,17 @@ func New(opts ...TrufflehogTransformerOption) (*trufflehogTransformer, error) {
 		return nil, err
 	}
 
+	stripFilePathPrefix, _ := env.GetOrDefault(
+		"TRUFFLEHOG_SOURCE_CODE_WORKSPACE",
+		"/workspace/source-code",
```

Comment:
I don't think we should have a default value over here, instead it should fail if this value is not provided